src/rest_framework_dso/fields.py

```diff
@@ -95,17 +96,16 @@ def is_array(self) -> bool:
 
     @cached_property
     def attname(self):
```

Comment:
Maybe add annotation for return type. Make it extra clear that it is heterogeneous.